Calendars/Calendars.Plugin.Android/DateConversions.cs

```diff
@@ -1,13 +1,15 @@
 ﻿using System;
 
+#nullable enable
+
 namespace Plugin.Calendars
 {
     /// <summary>
     /// Date conversion helpers.
     /// </summary>
     internal static class DateConversions
     {
-        private readonly static DateTime _reference = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
+        private readonly static DateTime _reference = new(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
Woah. I didn't know about this...

Reply:
This change doesn't alter any behavior (or even resulting IL code). Are you against C# 9 `new` syntax, or is there some other issue?

Reply:
No issue at all! I'm impressed. I didn't know about this syntax.

Reply:
I mean, this syntax was designed exactly for this: not having to duplicate type when assigning default value to properly or field

Reply:
Yep, makes perfect sense.

Reply:
Sorry, misread your first comment 😄
Yeah, that's a cool one. Basically you can now write `new()` (without type) if type can be inferred by the compiler. So, for example `new List<Point> { new(1, 2) }` is also valid, or `Point p = new(1, 2)`